dts/bindings/sensor/st,stm32-qdec.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2022, Valerio Setti <vsetti@baylibre.com
+# SPDX-License-Identifier: Apache-2.0
+
+description: STM32 quadrature decoder
+
+compatible: "st,stm32-qdec"
+
+include:
+    - name: base.yaml
+    - name: pinctrl-device.yaml
+
+properties:
+     pinctrl-0:
+       required: true
+
+     pinctrl-names:
+       required: true
+
+     input-polarity-inverted:
+       type: boolean
+       required: false
+       description: Encoder is triggered by a falling edge on the input pin
+
+     input-filter-level:
+       type: int
+       required: true
+       description: |
+         Intensity of the filter applied to the input signal. This is
+         implemented by scaling the sampling frequency and adding a counter
+         in which N consecutive samples with same value are needed to validate
+         a transition.
+         Mapping is as follows (F_clk is the timer's clock):
+         0: No filter, sampling is done at F_dts
+         1: Fs = F_clk, N=2
+         2: Fs = F_clk, N=4
+         3: Fs = F_clk, N=8
+         4: Fs = F_clk/2, N=6
+         5: Fs = F_clk/2, N=8
+         6: Fs = F_clk/4, N=6
+         7: Fs = F_clk/4, N=8
+         8: Fs = F_clk/8, N=6
+         9: Fs = F_clk/8, N=8
+         10: Fs = F_clk/16, N=5
+         11: Fs = F_clk/16, N=6
+         12: Fs = F_clk/16, N=8
+         13: Fs = F_clk/32, N=5
+         14: Fs = F_clk/32, N=6
+         15: Fs = F_clk/32, N=8
+       default: 0
```

Comment:
defaults need to be justified per dt guidelines

Reply:
At least on ST devices the answer should be yes, but since I agree this is not general and also according to the previous comment, I removed this part

Reply:
There is an explanation of what the default value is, but not why it was chosen.